### PR TITLE
Allow graceful handling of CADogtagCertsConfigCheck with no certs

### PR DESCRIPTION
**Pull request by ckelleyRH**
Currently if no cert is returned from the nssdb with the requested nickname the test tries to process the NoneType cert and throws a TypeError, check explicitly if the cert is not None before proceeding with the rest of the test.

Resolves #2182990

### CODE REVIEW

File: base/server/healthcheck/pki/server/healthcheck/meta/csconfig.py
Comment (edewata) [inline base/server/healthcheck/pki/server/healthcheck/meta/csconfig.py:59]: This probably should be:
```
if cert_nssdb is None:
```

File: base/server/healthcheck/pki/server/healthcheck/meta/csconfig.py
Comment (edewata) [inline base/server/healthcheck/pki/server/healthcheck/meta/csconfig.py:64]: We probably should include the token name (if available) since IPA is planning to support HSM, so it should be just `<nickname>` if there's no token, or `<token>:<nickname>` if there's a token.

**Review by edewata — APPROVED** [review]
I have some comments, but in general it LGTM. Feel free to update & merge.

**Review by fmarco76 — APPROVED** [review]
Overall looks good. After fix, can be merged

File: base/server/healthcheck/pki/server/healthcheck/meta/csconfig.py
Comment (ckelleyRH) [reply base/server/healthcheck/pki/server/healthcheck/meta/csconfig.py]: Fixed.

File: base/server/healthcheck/pki/server/healthcheck/meta/csconfig.py
Comment (ckelleyRH) [reply base/server/healthcheck/pki/server/healthcheck/meta/csconfig.py]: Good idea, I added the token name in conditionally if it exists.

**Comment by ckelleyRH** [comment]
Thanks @edewata @fmarco76 !

**Pull request merged**
